### PR TITLE
Feature/#90 pwa 

### PR DESCRIPTION
**Pull request by flxtagi**
Screenshot 2023-04-13 at 16 25 35

looks neat! 😄 
works only with the last next js version

### CODE REVIEW

**Comment by rudigier** [comment]
Bildschirmfoto 2023-04-13 um 20 00 04

File: package.json
Comment (rudigier) [inline package.json:1]: Are there other impacts when change the next version?

**Review by rudigier — CHANGES_REQUESTED** [review]

**Comment by rudigier** [comment]
offline nothing get's displayed. Or do i something wrong?

**Comment by rudigier** [comment]
Bildschirmfoto 2023-04-13 um 20 13 31

File: package.json
Comment (flxtagi) [reply package.json:1]: lets check that by merging master and develop branch into it

File: public/sw.js
[the remark this answers is not in the document]
Comment (flxtagi) [reply public/sw.js:1]: removed

File: src/pages/_app.tsx
[the remark this answers is not in the document]
Comment (flxtagi) [reply src/pages/_app.tsx:1]: we should move assets to /assets folder. best practice

**Comment by flxtagi** [comment]
works now with next v.13.2.4
plaease check in on vercel on mobile

File: src/pages/_app.tsx
[the remark this answers is not in the document]
Comment (rudigier) [reply src/pages/_app.tsx:1]: Created a ticket https://github.com/smartive-education/app-pizza-hawaii/issues/105

**Review by rudigier — APPROVED** [review]

**Pull request merged**